packages/cadl-ranch-specs/http/models/visibility/main.cadl

```diff
@@ -0,0 +1,83 @@
+import "@cadl-lang/rest";
```

Comment:
Think it looks good. A good way to check you have the visibility setup correctly is to write the code with the openapi3 emitter(or in the cadl-playground)

https://cadlplayground.z22.web.core.windows.net/?c=aW1wb3J0ICJAY2FkbC1sYW5nL3Jlc3QiOwoKdXNpbmcgQ2FkbC5IdHRwOwoKCkBkb2MoIk91dHB1dCBtb2RlbCB3aXRoIHZpc2liaWxpdHkgcHJvcGVydGllcy4iKQrGJFbJH03FEHsKICDGS1JlcXVpcmVkIHN0cmluZywgaWxsdXN0cmF0xHxhIHJlYWRvbmx5IHZhbHVlIHR5cGXIYnnEYCAgQMp8KCLEL8UWcsdYUsdAU8VfOsdnOwryAIFpbnTRfndyaXRl33%2FMf8Uw7QCAV8hCSW50OsRkMzLUfWNyZWHHauYAoiBjb2xsZWN0aW9u02jGL81pQ8lC5gDrTGlzdOgA71td1HR1cGTIdGludN5xxizNcVXJP0ludMZu5QDfxG19CgpAZ2V0Cm9wIGdldOUB9CgpOvACBzsKCkBwdcUmcHXHJkBib2TkAJBwdXTRMCk6IHZvaWTFN2F0Y2jFOcQJ3zvPO29zxnNvc990zTk%3D

Reply:
There is some docs on the automatic visibility https://microsoft.github.io/cadl/standard-library/rest/operations#automatic-visibility

Would also maybe name the namespace `Models.Visibility.Automatic`

Reply:
data looks great now, think just can play a bit with the names to have the scenarios be better